src/Aquifer.AI/OpenAiTranslationService.cs

```diff
@@ -0,0 +1,194 @@
+﻿using System.Text;
+using System.Text.RegularExpressions;
+using Aquifer.Common.Clients;
+using Aquifer.Common.Utilities;
+using OpenAI.Chat;
+
+namespace Aquifer.AI;
+
+public interface ITranslationService
+{
+    /// <summary>
+    /// Translates plain text.
+    /// Note: It's up to the caller to guarantee that the text length is not too long.
+    /// </summary>
+    public Task<string> TranslateTextAsync(
+        string text,
+        (string Iso6393Code, string EnglishName) destinationLanguage,
+        CancellationToken cancellationToken);
+
+    /// <summary>
+    /// Translates HTML content while preserving the HTML tags.
+    /// Note: Translations are done on each individual paragraph in order to avoid hitting length limitations.
+    /// </summary>
+    public Task<string> TranslateHtmlAsync(
+        string html,
+        (string Iso6393Code, string EnglishName) destinationLanguage,
+        CancellationToken cancellationToken);
+}
+
+public class OpenAiOptions
+{
+    public required string Model { get; init; }
```

Comment:
The OpenAI client requires specifying the AI model in the constructor but we weren't previously passing the model in our homegrown client.  I'm assuming the model is `gpt-4o`.